### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by julialbq**
operations function is not working yet - it is called but doesn't work

### CODE REVIEW

File: global.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  }
+  
+  header {
+    display: flex;
+    place-items: center;
+    gap: 16px;
+    margin-bottom: 16px;
+    box-shadow: 0px 0px 5px 6px #ccc;
+  }
+
```
Comment (jesrodri) [inline global.css]: :nail_care: last line should be empty

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                <p class="display"></p>
+            </div>
+            <button class="button clean" onclick="cleanDisplay()">C</button>
+        </div>
+        <div class="buttons">
+                <button class="button number" onclick="displayValue('7')">7</button>
+                <button class="button number" onclick="displayValue('8')">8</button>
+                <button class="button number" onclick="displayValue('9')">9</button>
+                <button class="button operation" onclick="displayValue('÷')">÷</button>
+                <button class="button umber" onclick="displayValue('4')">4</button>
```
Comment (jesrodri) [inline index.html]: typo: `class="button umber"`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                <button class="button operation" onclick="displayValue('-')">-</button>   
+                <button class="button number" onclick="displayValue('0')">0</button>
+                <button class="button number" onclick="displayValue('.')">.</button>
+                <button class="button operation" onclick="displayValue('+')">+</button>
+                <button class="button equal" onclick="displayResult()">=</button>  
+        </div>
+    </section>
+    <script src="./index.js"></script>
+</body>
+</html>
```
Comment (jesrodri) [inline index.html]: :nail_care: empty line at eof

File: style.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.equal, .clean {
+    background-color: #6F8FAF;
+    color: white;
+}
+
+.operation {
+    background-color: #DCD2CC;
+}
```
Comment (jesrodri) [inline style.css]: :nail_care: empty line at eof

File: index.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            <div>
+                <p class="display"></p>
+            </div>
+            <button class="button clean" onclick="cleanDisplay()">C</button>
+        </div>
+        <div class="buttons">
+                <button class="button number" onclick="displayValue('7')">7</button>
+                <button class="button number" onclick="displayValue('8')">8</button>
+                <button class="button number" onclick="displayValue('9')">9</button>
+                <button class="button operation" onclick="displayValue('÷')">÷</button>
```
Comment (jesrodri) [inline index.html]: be careful with the chars! `eval` does not recognize `÷` as a valid operation

File: global.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
+body {
+    margin: 0;
```
Comment (jesrodri) [inline global.css]: :nail_care: identation should have only 2 spaces

File: index.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (jesrodri) [inline index.html]: :nail_care: identation should have only 2 spaces

File: index.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
+const display = document.querySelector('.display')
+
+function cleanDisplay() {
+    display.innerText = ""
```
Comment (jesrodri) [inline index.js]: :nail_care: identation should have only 2 spaces

File: style.css [hunk 9]
```diff
@@ -0,0 +1,64 @@
+header {
+    text-align: center;
```
Comment (jesrodri) [inline style.css]: :nail_care: identation should have only 2 spaces

File: style.css [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+    justify-content: space-between;
+    width: 100%;
+}
+
+.screen > div {
+    background-color: #E8E8E8;
+    border: 1px solid #B9B7BD;
+    border-radius: 8px;
+    width: 85%;
+}
```
Comment (jesrodri) [inline style.css]: suggestion: add `padding` in this `div` to avoid text content from being too close to the border

File: global.css [hunk 11]
```diff
@@ -1,17 +1,20 @@
...
+  display: flex;
+  place-items: center;
+  gap: 16px;
+  margin-bottom: 16px;
+  box-shadow: 0px 0px 5px 6px #ccc;
+}
+
+p {
+  margin: 0;
+}
```
Comment (jesrodri) [inline global.css]: :nail_care: empty line

File: index.js [hunk 12]
```diff
@@ -1,20 +1,54 @@
...
+  }
 }
 
 function displayResult() {
-    let operation = display.innerText
-    let result = eval(operation)
+  let operation = display.innerText
+  let result = eval(operation)
 
+  let correctDecimals = countDecimals(result)
```
Comment (jesrodri) [inline index.js]: we can improve this variable's name. what do you think about `decimalPlaces` ?

Comment (julialbq) [reply index.js]: Much better!

File: index.html [hunk 13]
```diff
@@ -1,46 +1,46 @@
...
-        </div>
-    </section>
-    <script src="./index.js"></script>
+  <header>
+      <h1>Calculator</h1>
+  </header>
+  <section class="container">
+    <div class="screen">
+      <div>
+        <p class="display"></p>
```
Comment (jesrodri) [inline index.html]: Classes are usually used for styling.
We can give this `p`tag an `id` and then call `getElementById` to find the element on the JavaScript code.

**Review by jesrodri — APPROVED** [review]

**Pull request merged**